util/bux/package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "build": "tsc -b ./config/tsconfig.build.json && node ./bin/bux chmodx ./bin/bux",
     "bux": "npm run build && node ./bin/bux",
-    "clean": "rimraf ./lib && rimraf ./build",
+    "clean": "rimraf ./lib && rimraf ./build && rimraf ./resources",
```

Comment:
It doesn't look like we need to clear out the resources directory here (at least I'm not seeing any resources directory being created when I build bux)